litex/soc/interconnect/axi.py

```diff
@@ -60,12 +60,12 @@ def __init__(self, data_width, address_width, id_width=1, clock_domain="sys"):
 
 class AXI2Wishbone(Module):
     def __init__(self, axi, wishbone, base_address):
-        assert axi.data_width    == 32
+        assert axi.data_width    == len(wishbone.dat_r)
         assert axi.address_width == 32
```

Comment:
maybe also replace "32" with "`assert axi.address_width == len(wishbone.adr) + 2`" ?  (to minimize hard-coded magic numbers)